Camera:
  Focal: 13.971313932573906
  High: 650.0
Center:
  Camera_high: 620
  Center_X: 333
  Center_Y: 255
  Front: 90
  Horizon: 188
  Inner: 51
  Outer: 280
  X: 308
  Y: 266
FPS: 16
HSV:
  Ball:
  - 0
  - 26
  - 30
  - 100
  - 18
  - 100
  Blue:
  - 224
  - 261
  - 68
  - 100
  - 40
  - 99
  Green:
  - 150
  - 199
  - 32
  - 86
  - 20
  - 90
  White:
  - 216
  - 168
  - 0
  - 71
  - 0
  - 29
  Yellow:
  - 39
  - 79
  - 10
  - 100
  - 56
  - 100
  black:
    angle: 1
    gray: 37
  white:
    angle: 3
    gray: 193
Parameterbutton: 4
SCAN:
  Angle_Near_Gap: 4
  Angle_range_1: 2
  Angle_range_2_3: 2
  Dont_Search_Angle_1: 31
  Dont_Search_Angle_2: 150
  Dont_Search_Angle_3: 268
  Magn_Far_End: 267
  Magn_Far_Start: 126
  Magn_Middle_Start: 50
  Magn_Near_Gap: 4
  Magn_Near_Start: 44
blackItem:
  angle: 3
  gray: 40
whiteline:
  angle: 4
  gray: 100
